RecoTracker/TkDetLayers/test/TkDetLayersAnalyzer.cc

```diff
@@ -2,24 +2,23 @@
 #include <memory>
 
 // user include files
-#include "FWCore/Framework/interface/Frameworkfwd.h"
-#include "FWCore/Framework/interface/one/EDAnalyzer.h"
+#include "DataFormats/GeometrySurface/interface/BoundSurface.h"
+#include "DataFormats/TrackerCommon/interface/TrackerTopology.h"
+#include "FWCore/Framework/interface/ESHandle.h"
```

Comment:
Doesn't seem to be needed any more

Reply:
removed in [0a9ee2d](https://github.com/cms-sw/cmssw/pull/36980/commits/0a9ee2def74a0cf443e5e22a5b52be489d5a03de)